cdap-notifications-api/src/main/java/co/cask/cdap/notifications/feeds/NotificationFeedNotFoundException.java

```diff
@@ -16,8 +16,10 @@
 
 package co.cask.cdap.notifications.feeds;
 
+import co.cask.cdap.proto.Id;
+
 /**
- * Exception thrown when a {@link NotificationFeed} object is not found.
+ * Exception thrown when a {@link Id.NotificationFeed} object is not found.
  */
 public class NotificationFeedNotFoundException extends NotificationFeedException {
```

Comment:
We have ``co.cask.cdap.common.NotFoundException``. Could you use that instead? For a later PR maybe.